src/PieChart.js

```diff
@@ -286,7 +293,7 @@ class PieChart extends React.Component {
       markerLineOverhangOuter,
     } = this.props;
 
-    const valueAccessor = makeAccessor(this.props.getValue);
+    const valueAccessor = makeAccessor(this.props.slice);
     const sum = sumBy(this.props.data, valueAccessor);
     const total = this.props.total || sum;
```

Comment:
same as above

Reply:
This requires renaming the variable `total` because total would then be defined twice. Will do it, just an FYI.